configuration/configuration.go

```diff
@@ -693,3 +714,31 @@ func Parse(rd io.Reader) (*Configuration, error) {
 
 	return config, nil
 }
+
+const (
+	// ExporterTypeOTLP represents the open telemetry exporter OTLP
+	// https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/otlp.md
+	ExporterTypeOTLP = "otlp"
+
+	// ServiceName is trace service name
+	ServiceName = "distribution"
+
+	// DefaultSamplingRatio default sample ratio
+	DefaultSamplingRatio = 1
+)
+
+// Validate OpenTelemetry config
+func (otel *OpenTelemetryConfig) Validate() error {
+	switch otel.ExporterName {
+	case ExporterTypeOTLP:
+		if otel.ServiceName == "" {
+			otel.ServiceName = ServiceName
+		}
+		if otel.TraceSamplingRatio == 0 {
+			otel.TraceSamplingRatio = DefaultSamplingRatio
+		}
+		return nil
+	default:
+		return errors.Wrapf(errors.New("invalid argument"), "unsupported exporter: %+v", otel)
```

Comment:
Wondering why this is using `errors.Wrapf()`; the error that's wrapped doesn't appear to have a type associated, so probably can't be used for detection here?

Reply:
Great catch, Seb! Yeah, we should just return `fmt.Errorf` here. No need to wrap anything since it's all plain strings.